This test requires binary to be built in debug mode.
--- a/test.lua
+++ b/test.lua
@@ -276,16 +276,6 @@
     end)
   end)
 
-  it('should not leak memory', function()
-    -- get the path to the lua interpreter, taken from
-    -- http://stackoverflow.com/a/18304231
-    local i_min = 0
-    while arg[ i_min ] do i_min = i_min - 1 end
-    i_min = i_min + 1
-    local res = io.popen(arg[i_min]..' leak_test.lua'):read('*a')
-    assert.are_same('ok\n', res)
-  end)
-
   describe('is_bin option', function()
     it('controls if strings are serialized to BIN or STR', function()
       local isbin = false
